.github/workflows/cli.yaml

```
@@ -39,15 +43,24 @@ jobs:
           version: "v0.22.0"
       - name: run karmadactl init test
         run: |
-          export CLUSTER_VERSION=kindest/node:${{ matrix.k8s }}
-
           # init e2e environment
           hack/cli-testing-environment.sh
 
           # run a single e2e
           export KUBECONFIG=${HOME}/karmada/karmada-apiserver.config
           GO111MODULE=on go install github.com/onsi/ginkgo/v2/ginkgo
           ginkgo -v --race --trace -p  --focus="[BasicPropagation] propagation testing deployment propagation testing"  ./test/e2e/
+      - name: check
+        if: ${{ matrix.extraEtcdVersion != '' }} 
+        run: |
+          kubectl --kubeconfig ~/.kube/karmada-host.config get pod -A -owide
+
+          # would not deploy etcd when useing extra etcd.
+          KARMADA_ETCD_STS_COUNT=`kubectl --kubeconfig ~/.kube/karmada-host.config get sts -n karmada-system -l karmada.io/bootstrapping=app-defaults  -ojson | jq '.items | length'`
```

```suggestion
          KARMADA_ETCD_STS_COUNT=`kubectl --kubeconfig ~/.kube/karmada-host.config get sts -n karmada-system -l karmada.io/bootstrapping=app-defaults,app=etcd  -ojson | jq '.items | length'`
```
label `karmada.io/bootstrapping=app-defaults` is used to represent karmada statefulSet deployment and can not quite match up to etcd